pkg/persistentvolumeclaim/debug_cstor.go

```diff
@@ -183,3 +181,25 @@ func resourceStatus(crs util.CstorVolumeResources) error {
 
 	return nil
 }
+
+func displayPVCEvents(k client.K8sClient, crs util.CstorVolumeResources) error {
```

Comment:
TODO: (later) This should be converted into a method and moved to `pkg/client` someday, I believe this was done because one cannot make it generic right now with kind missing.

Reply:
This method wont be useful for any other volume/cas-type and thus made it unexported and kept in this package.

Reply:
No I mean this function code would have to be copied multiple times to different places to be used, which isn't something desirable, I'll ask around if this problem is fix-able somehow by getting the kind correctly.